scripts/patterns-reorg/models.py

```diff
@@ -109,14 +115,33 @@ def filter_entries(
         result.append(new_entry)
     return tuple(result)
 
+def add_redirect_to_dict(dict: dict[str, str], old_url: str, redirect_to: str)-> None:
```

Comment:
`dict` is a dangerous name because you're shadowing the builtin value `dict`

Reply:
Also it'd be good to put a `_` in front of the function name since this is a private helper